profiles/coreos/base/profile.bashrc.slsa-provenance

```diff
@@ -0,0 +1,194 @@
+            fi
+        done
+    elif [ -n "${EGIT_REPO_URI:-}" ] ; then
+        # package is built from repo checkout (git)
+        einfo "    Provenance: recording GIT material (input) '${EGIT_REPO_URI}'"
+        csum="${EGIT_COMMIT}"
+        uri="${EGIT_REPO_URI}"
+        echo -e ",\n      { \"uri\": \"${uri}\","
+        echo -n "        \"digest\": {\"sha1\":\"$csum\"} }"
+    fi
+
+    # Patches / files shipped with the ebuild (if any)
+    csum="$(cat "/mnt/host/source/src/scripts/.git/modules/sdk_container/src/third_party/${repo}/HEAD")"
+    uri="git+https://github.com/flatcar-linux/${repo}.git@${csum}#${CATEGORY}/${PN}/files"
+    if [ -d "${FILESDIR}" ] ; then
+        for file in $(cd "$FILESDIR" && find . -type f | sed 's:^./::') ; do
+            csum="$(sha1sum - <"${FILESDIR}/${file}")"
+            csum="${csum%% *}"
+            einfo "    Provenance: recording ebuild material (input) '${file}'"
+            echo -e ",\n      { \"uri\": \"${uri}/${file}\","
+            echo -n "        \"digest\": {\"sha1\":\"$csum\"} }"
+        done
+    fi
+
+    echo -ne '\n   ]'
+}
+# --
+
+__slsa_provenance_report() {
+    local scripts_hash="$(cat "/mnt/host/source/src/scripts/.git/HEAD")"
+    local buildcmd="emerge"
+    # extract board from e.g. '/build/amd64-usr/build'. Empty if no board is set (SDK build).
+    local board="$(echo "${CROS_BUILD_BOARD_TREE:-}" | sed -n 's:^/build/\([^/]\+\)/.*:\1:p')"
+    if [ -n "$board" ] ; then
+        buildcmd="emerge-${board}"
+    fi
+    if [[ "${scripts_hash}" == "ref:"* ]]; then
+        scripts_hash="$(cat /mnt/host/source/src/scripts/.git/${scripts_hash#ref: })"
+    fi
+
+    # FIXME: Supply SDK image ID and sha256 digest along with the version tag
+    local sdk_version="$(source /mnt/host/source/.repo/manifests/version.txt; echo ${FLATCAR_SDK_VERSION})"
+
+    # FIXME: add builder ID
+cat <<EOF
+{
+ "_type": "https://in-toto.io/Statement/v0.1",
+ "predicateType": "https://slsa.dev/provenance/v0.2",
+ "predicate": {
+   "buildType": "ghcr.io/flatcar-linux/flatcar-sdk-all:${sdk_version}",
+   "builder": {"id": "TODO - builder ID" },
+   "invocation": {
+      "configSource": {
+        "uri": "https://github.com/flatcar-linux/scripts",
+        "digest": {"sha1": "${scripts_hash}"}
+      }
+   },
+   "buildConfig": {
+     "commands": [
+        "git checkout ${scripts_hash}",
+        "git submodule init",
+        "git submodule update",
+        "./run_sdk_container ${buildcmd} =${CATEGORY}/${PF}"
+    ]
+   },
+EOF
+    __slsa_provenance_materials
+    echo ","
+    __slsa_provenance_subject
+    echo ""
+cat <<EOF
+  }
+}
+EOF
+}
+# --
+
+cros_post_src_install_generate_slsa_provenance_report() {
+    if [ "${generate_slsa_provenance_report:-no}" != "yes" ] ; then
+        return
+    fi
+
+    local report_file="${CATEGORY}_${PF}.json.bz2"
+    local dest_dir="${D}/usr/share/SLSA/"
+
+    __slsa_provenance_report | lbzip2 -9cz > "${T}/${report_file}"
```

Comment:
Maybe use jq to format the report nicely:
```suggestion
    __slsa_provenance_report | jq | lbzip2 -9cz > "${T}/${report_file}"
```

Reply:
done